subprojects/gradle/tms/src/main/kotlin/com/avito/plugin/MarkReportAsSourceAction.kt

```diff
@@ -25,38 +25,31 @@ class MarkReportAsSourceAction(
     fun mark(reportCoordinates: ReportCoordinates) {
         val testSuiteVersion = timeProvider.nowInMillis()
 
-        reportsApi.getReport(reportCoordinates).fold(
-            onSuccess = { report ->
+        reportsApi.getReport(reportCoordinates)
+            .map { report ->
```

Comment:
need to pass `report` through the chain, that's why it's `map` instead of `flatMap`